modules/libcom/src/error/errlog.h

```diff
@@ -275,13 +275,15 @@ LIBCOM_API void errSymLookup(long status, char *pBuf, size_t bufLength);
 #define ANSI_ESC_BLUE "\033[34;1m"
 #define ANSI_ESC_MAGENTA "\033[35;1m"
 #define ANSI_ESC_CYAN "\033[36;1m"
+#define ANSI_ESC_BOLD "\033[1m"
 #define ANSI_ESC_RESET "\033[0m"
 #define ANSI_RED(STR)     ANSI_ESC_RED     STR ANSI_ESC_RESET
 #define ANSI_GREEN(STR)   ANSI_ESC_GREEN   STR ANSI_ESC_RESET
 #define ANSI_YELLOW(STR)  ANSI_ESC_YELLOW  STR ANSI_ESC_RESET
 #define ANSI_BLUE(STR)    ANSI_ESC_BLUE    STR ANSI_ESC_RESET
 #define ANSI_MAGENTA(STR) ANSI_ESC_MAGENTA STR ANSI_ESC_RESET
 #define ANSI_CYAN(STR)    ANSI_ESC_CYAN    STR ANSI_ESC_RESET
+#define ANSI_BOLD(STR)    ANSI_ESC_BOLD    STR ANSI_ESC_RESET
```

Comment:
Future enhancement: There is another code `"\033[22m"` which clears bold mode without also affecting the foreground color. Using that instead of the reset sequence would allow nesting of `ANSI_BOLD()` with the colors.

Reply:
Nesting seems interesting, but something I'll leave for the future.  Note that the `ANSI_ESC_*` color code macros currently set both foreground color and bold (eg. `"\033[31;1m"`).  So as is, nesting an SGR 22 would have the effect to un-bold the following text.